docs/.vitepress/config.ts

```diff
@@ -93,66 +93,92 @@ export default defineConfig({
+              collapsed: true,
               items: [
+                { text: 'Create an account', link: '/developers/guides/advanced/register-a-user' },
+                { text: 'Create an account key', link: '/developers/guides/advanced/add-signer' },
+                { text: 'Find account by name', link: '/developers/guides/advanced/name-lookup' },
+                { text: 'Change farcaster name', link: '/developers/guides/advanced/transfer-fname' },
+                { text: 'Change custody address', link: '/developers/guides/advanced/transfer-fid' },
+                { text: 'Change recovery address', link: '/developers/guides/advanced/change-recovery' },
+                // { text: 'Rotate an account key', link: '/developers/guides/applications/todo' },
+              ],
+            },
+            {
+              text: 'Querying data',
+              collapsed: true,
+              items: [
+                // { text: 'Get account messages', link: '/developers/guides/advanced/todo' },
+                // { text: 'Get account profile', link: '/developers/guides/advanced/todo' },
                 {
-                  text: 'Connecting to Hubble',
+                  text: 'Query Hubble',
                   link: '/developers/guides/basics/setting-up',
                 },
-                { text: 'Replicate to postgres', link: '/developers/guides/basics/replicate' },
-                { text: 'Create an Account', link: '/developers/guides/basics/account' },
+                { text: 'Fetch channel casts', link: '/developers/guides/advanced/fetch-channel-casts' },
               ],
             },
-            // {
-            //   text: 'Applications',
-            //   collapsed: false,
-            //   items: [
-            //     { text: 'Generate a chronological feed for a user', link: '/developers/guides/applications/feed' },
-            //     { text: 'Create a bot to post messages to the hub', link: '/developers/guides/applications/bot' },
-            //     // { text: 'Build a farcaster client', link: '/developers/guides/applications/client' },
-            //     // { text: 'Read data with Farcaster Connect', link: '/developers/guides/applications/todo' },
-            //     // { text: 'Write data with Farcaster Connect', link: '/developers/guides/applications/todo' },
-            //   ],
-            // },
             {
-              text: 'Advanced',
+              text: 'Writing data',
+              collapsed: true,
+              items: [
+                { text: 'Create a cast', link: '/developers/guides/advanced/adv-casts' },
+                { text: 'Create a channel cast', link: '/developers/guides/advanced/create-channel-casts' },
+                // Rather than separate topics we might want to consider a page for  a topic "e.g. casts"
+                // which covers adding and deleting all kinds.
+                //
+                // { text: 'Remove a cast', link: '/developers/guides/advanced/todo' },
```

Comment:
@sanjayprabhu added a few more "basic tutorials" that I think would be helpful for people to get oriented